README.md

```diff
@@ -13,6 +13,16 @@ If you do want to allow access from all remotes, start ark-rpc with the `--allow
 - install ark-rpc: `npm install arkecosystem/ark-rpc#master`
 - start RPC server: `ark-rpc --port 8000` (default port is 8080)
 
+## Docker ##
+If you would like to run from a docker environment, you will first need to build the container by running:
+```
+docker build -t ark-rpc .
+```
+You will need to run the container with the `--allow-remote` option to allow the host machine to access the container.
+```
+docker run -d -p 8080:8080 ark-rpc --allow-remote
```

Comment:
I'm not too familiar with docker, but I can see you gave port 8080 here, whereas the dockerfile has 8081. Is that intentional/expected or have I misunderstood?

Reply:
Ah, Good catch Alex. EXPOSE doesn't actually publish any ports, so I missed that when I was testing. Just pushed the fix.